javascript-forms-starter-files/starter-files/form-events.js

```diff
@@ -0,0 +1,29 @@
+function handleFocus(event){
+  console.log('focus event was fired')
+  console.log('event.target.name')
```

Comment:
looks like you forgot to actually log out the `event.target.name` - what you have is just the label